runtime/interpreter/encoding_test.go

```diff
@@ -389,6 +408,8 @@ func TestEncodeDecodeUint64AtreeValue(t *testing.T) {
 
 func TestEncodeDecodeArray(t *testing.T) {
 
+	t.Skip("skipping ArrayValue encoding and decoding test because it involves implementation details in atree repo")
```

Comment:
Should this test and `TestEncodeDecodeComposite` be adjusted and unskipped before merging the PR / is this a TODO?

Reply:
Although we can adjust these two tests, I am not sure if we should.

`TestEncodeDecodeArray` encodes `atree.Array.Storable()`.  Encode function can fail if array is inlined because we can't encode inlined array by itself.  So we can either:
- create a large atree.Array under the hood (larger than max size limit), so `Storable()` returns a `SlabID`  like before, or 
- encode slab containing array and then test encoded bytes.  

Either way, these two approaches depend on implementation details in atree (any encoding changes to atree array/map would break these tests).  These two tests are different from other encoding tests because others test encoding defined in Cadence.

Thoughts?